src/fit/FitPathProc.hh

```diff
@@ -0,0 +1,83 @@
+#ifndef __RAT_FitPathProc__
+#define __RAT_FitPathProc__
+
+#include <string>
+#include <vector>
+#include <RAT/SimulatedAnnealing.hh>
+#include <RAT/Processor.hh>
+
+
+#include <Minuit2/FCNBase.h>
+
+using namespace ROOT::Minuit2;
```

Comment:
I am wary of `using`s in headers (even though this case looks relatively harmless) -- could you move this to the source file?